tests/macros/m18235.nim

```diff
@@ -4,7 +4,7 @@ import macros
 # processed by a type macro. Used by a test of a corresponding name of this
 # file.
 
-macro rename(n: typed): untyped =
+macro eexport(n: typed): untyped =
```

Comment:
reexport?

Reply:
"export" makes more sense than "reexport" so lets keep it as is, just a test anyways